src/lib/rule-finder.js

```diff
@@ -58,7 +58,7 @@ function _getPluginRules(config) {
       var normalized = _normalizePluginName(plugin)
       var pluginConfig = require(normalized.module)
 
-      var rules = pluginConfig.rules
+      var rules = pluginConfig.rules || {}
```

Comment:
[I personally stopped using `||` for setting default value.](https://sarbbottam.github.io/javascript/2016/05/27/never-use-or-for-default-values/)
I would use 
```
var rules = pluginConfig.rules === undefined ? {} : pluginConfig.rules
```